src/main/scala/com/fulcrumgenomics/fastq/DemuxFastqs.scala

```diff
@@ -298,19 +319,46 @@ class DemuxFastqs
  @arg(doc="The sequencing center from which the data originated") val sequencingCenter: Option[String] = None,
  @arg(doc="Predicted median insert size, to insert into the read group header") val predictedInsertSize: Option[Integer] = None,
  @arg(doc="Platform model to insert into the group header (ex. miseq, hiseq2500, hiseqX)") val platformModel: Option[String] = None,
+ @arg(doc="Platform to insert into the group header (e.g Illumina)") val platform: String = "Illumina",
```

Comment:
```suggestion
 @arg(doc="Platform to insert into the read group header of BAMs (e.g Illumina)") val platform: String = "Illumina",
```

Also aren't the platforms supposed to be all upper case?